model: farryn
M1_P: 8908.51572
M2_P: 9960.05140
M1_I: 1038.91467
M2_I: 490.19002
M1_QPPS: 6432
M2_QPPS: 6116
AXLE_WIDTH: 0.188
motor_usb_port: /dev/roboclaw

KP_velocity: 15
frw_vel_topic: /summit_xl/joint_frw_velocity_controller/command

#cmd_vel_topic: /robot_cmd_vel #/cmd_vel_mux/input/teleop
cmd_vel_topic: /cmd_vel_mux/input/teleop
